app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -42,6 +42,19 @@ def host
           end
         end
 
+        api :GET, '/subnets/:subnet_id', N_('Download subnet image')
+        param :subnet_id, :identifier_dottable, :required => true
+        def subnet
+          id = params[:id]
+          subnet = Subnet.find(id) || render_message(_("Cannot find subnet with id %s") % id) && return
```

Comment:
This controller's scoped to managed hosts, perhaps put it into another controller and ensure it's scoped correctly?